crates/composition/src/subgraphs.rs

```diff
@@ -66,22 +67,20 @@ impl Subgraphs {
     /// and field name. The argument is a closure that receives, for each group, the first field of
     /// the group and an iterator over all the other fields in the group. The order of iteration is
     /// deterministic but unspecified.
```

Comment:
This comment doesn't look correct anymore

Reply:
True, good catch, fixing it.